sdk/keyvault/keyvault-admin/test/public/accessControlClient.spec.ts

```diff
@@ -138,7 +138,13 @@ describe("KeyVaultAccessControlClient", () => {
 
     describe("upsertRoleDefinition", function() {
       it("errors when name is not a valid guid", async function() {
-        await assert.isRejected(client.upsertRoleDefinition(globalScope, "foo", []));
+        await assert.isRejected(
+          client.upsertRoleDefinition(globalScope, {
+            roleDefinitionName: "foo",
+            roleName: "foo",
```

Comment:
Nit (and below): I'd make these different just to reinforce in test data they are different values and were accidentally transposed or anything.

Reply:
Can do! Thank you!